backend/lib/endpoints/schema/posts.js

```diff
@@ -60,6 +61,7 @@ const updatePostSchema = {
         .prop("playStore", S.string().format("url"))
         .prop("website", S.string().format("url")),
     )
+    .prop("isEdited", S.boolean())
```

Comment:
You don't need the user to tell you if the post was edited, you just set it to true on the post update endpoint

Reply:
Thanks @Zaydme !